check_replication_queue.py

```diff
@@ -0,0 +1,39 @@
+import sys
+import waiting
+import click
+import logging
+import requests
+
+DEFAULT_TIMEOUT_THRESHOLD = 300
+DEFAULT_POLL_PERIOD = 60
+
+logging.basicConfig()
+logger = logging.getLogger("replication_queue")
+logger.setLevel(logging.INFO)
+
+
+def _is_replication_queue_empty(host: str, port: str):
```

Comment:
This does not seem right. It is not deterministic that replication queue being empty is a good signal. I would recommend that you get the metric being fetched in this datadog graph https://app.datadoghq.com/dashboard/mb4-iya-qa7/click-house?fullscreen_end_ts=1710785916256&fullscreen_paused=false&fullscreen_refresh_mode=sliding&fullscreen_section=overview&fullscreen_start_ts=1710771516256&fullscreen_widget=303213365&refresh_mode=sliding&view=spans&live=true to build the feature that replication has caught up.

Replication keeps on happening on clickhouse clusters all the time. What we are more interested in is how much is it lagging behind. You can try this out yourself. Go to snuba admin and run the command on any of the hosts of clickhouse cluster. You most likely will see some replication queue entries in there.

Reply:
Do we want[ replication queue inspection](https://www.notion.so/sentry/Rachel-30-60-90-Plan-ec06c00fd59c4493a997dcfa06d67548?pvs=4#7c7ab93d480c4a0ea64f2f8a192b4e66) to be a separate feature?

Reply:
There isn't high value in inspecting the replication queue by itself. It is not an appropriate signal to monitor. So I would say that it is not needed.